drv/gimlet-seq-server/src/main.rs

```diff
@@ -111,11 +121,16 @@ enum Trace {
     SpdBankAbsent(u8),
     SpdAbsent(u8, u8, u8),
     SpdDimmsFound(usize),
-    I2cFault {
-        retries_remaining: u8,
+    I2cError {
+        txn: I2cTxn,
         #[count(children)]
         code: i2c::ResponseCode,
     },
+    I2cFault(I2cTxn),
+    I2cRetry {
+        txn: I2cTxn,
```

Comment:
Maybe worth adding 
```suggestion
       #[count(children)]
        txn: I2cTxn,
```
here though, so that we can see the breakdown of retries by transaction that was retried?

N.B. that counting variants of the `I2cFault` doesn't seem necessary since it's fatal and we only record one of them, and it will almost certainly still be in the ringbuf if we ever record it...

Reply:
Good idea; fixed.